scripts/release/release.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/sh
 
 # do we have enough arguments?
-if [ $# < 3 ]; then
+if [ $# < 4 ]; then
     echo "Usage:"
     echo
-    echo "./release.sh <release version> <development version>"
+    echo "./release.sh <release version> <development version> <milestone id>"
     exit 1
 fi
 
 # pick arguments
 release=$1
 devel=$2
+milestone=$3
 
 # get current branch
 branch=$(git status -bs | awk '{ print $2 }' | awk -F'.' '{ print $1 }' | head -n 1)
 
-# manually edit and commit changelog changes
-# see https://github.com/bigdatagenomics/adam/issues/936
-#./scripts/changelog.sh $1 | tee CHANGES.md
-#git commit -a -m "Modifying changelog."
+# update changelog per Github milestone
+mvn com.github.heuermh.maven.plugin.changes:github-changes-maven-plugin:1.0:github-changes -DmilestoneId=${milestone}
```

Comment:
Is this your own plugin?

Reply:
Yeah, so if specifying the Github milestoneId directly isn't the right interface, I could change it to do something else.

Reply:
Ah ok